pillar/consul/init.sls

```diff
@@ -21,6 +22,10 @@ consul:
         node_ttl: 30s
         service_ttl:
           "*": 30s
+      {% if encrypt_key %}
+      encrypt: {{ encrypt_key }}
+      {% else %}
       encrypt: __vault__::secret-operations/global/consul-shared-secret>data>value
+      {% endif %}
```

Comment:
Should we flip the logic here so that normal deployments use vault since bootstrap is an exception?

Reply:
So, the default is that it will use the value from Vault as the SDB contents will only be present if we set it manually when rendering pillar data.